hypertrace-gradle-publish-maven-central-plugin/src/test/java/org/hypertrace/gradle/publishing/PublishMavenCentralPluginTest.java

```diff
@@ -20,7 +20,7 @@ public static void setup() {
     project.getPluginManager().apply("org.hypertrace.publish-maven-central-plugin");
     project.getExtensions().getByType(HypertracePublishMavenCentralExtension.class).license.set(License.APACHE_2_0);
     project.getExtensions().getByType(HypertracePublishMavenCentralExtension.class).repoName.set("test");
-    project.getTasksByName("tasks", false);
+    project.getAllTasks(true); // FIXME: why is this failing - something with kotlin dsl
```

Comment:
This one's weird, I think it's something like a difference in the third-party's kotlin version vs our own.